ibm-wh-acd-ce/ibm-wh-acd-ce/crtg/chart/crtg/values.yaml

```diff
@@ -18,16 +18,16 @@ configurationStorage:
     volume:
       existingClaimName: ""
       supplementalGroup: ""
-digest: "a1a14c28a4133625cd9f845bb2e111b5c37fb24155dd1f07bfc1d2609b334d1e"
-image: "cp.icr.io/cp/wh-acd/cartridge"
+container:
+  digest: "a1a14c28a4133625cd9f845bb2e111b5c37fb24155dd1f07bfc1d2609b334d1e"
+  image: "cp.icr.io/cp/wh-acd/cartridge"
+  tag: "20220613.1334"
 initContainer:
   digest: "6bcad846562eb85c16ad9e2f35f524d5e53dd88df29490436383fbcc29ad95b9"
   image: "cp.icr.io/cp/wh-acd/ubi8-whcs-liberty"
 revProxy:
-  digest: "fd2093a24bb7dcac048d0adc19ac1bc81219405bfbcd67b35858c0902b1d124f"
-  image: "nginxinc/nginx-unprivileged"
+  image: us.icr.io/vdt-acd-rns/common-services/nginxc-unpriveleged:stable
```

Comment:
Do we have and use the nginx image elsewhere? I'm wondering if we shouldn't be using the one from dockerhub instead? https://hub.docker.com/r/nginxinc/nginx-unprivileged. Not sure who maintains the stable version we have in VDT or how that one is updated?